.github/workflows/ci.yaml

```diff
@@ -42,8 +42,7 @@ jobs:
 
       - name: test
         run: |
-          flutter pub run test_coverage --no-badge
-          flutter pub run remove_from_coverage -f coverage/lcov.info -r ".g.dart$" -r ".freezed.dart$"
```

Comment:
I can implement this manually if you really need. But do you really need it?

Reply:
We can use `flutter test --coverage` instead of `flutter pub run test_coverage --no-badge`. Then `test_coverage` package will not be needed. Line with `remove_from_coverage` can be used as previously.